packages/react-app/src/components/ChallengeExpandedCard.jsx

```diff
@@ -133,92 +139,162 @@ const ChallengeExpandedCard = ({
   }
 
   return (
-    <Flex maxW={880} borderWidth="1px" borderRadius="lg" borderColor={borderColor} overflow="hidden" mb={6}>
-      <Center borderBottom="1px" borderColor={borderColor} w="200px" flexShrink={0} p={1}>
-        {challenge.previewImage ? (
-          <Image src={challenge.previewImage} objectFit="cover" />
-        ) : (
-          <Text p={3} textAlign="center">
-            {challengeId} image
-          </Text>
-        )}
-      </Center>
-      <Flex pt={6} pb={4} px={4} direction="column" grow={1}>
-        <Flex justify="space-between" pb={4}>
-          <Text fontWeight="bold">{challenge.label}</Text>
-          {challengeStatus && (
-            <Badge borderRadius="xl" colorScheme={colorScheme} textTransform="none" py={0.5} px={2.5}>
-              {label}
-            </Badge>
-          )}
-        </Flex>
-        <Text color={secondaryFontColor} mb={4}>
-          {challenge.description}
-        </Text>
-        <Spacer />
-        {challenge.externalLink?.link ? (
-          // Redirect to externalLink if set (instead of challenge detail view)
-          <ButtonGroup>
-            <Button
-              as={isChallengeLocked ? Button : Link}
-              href={isChallengeLocked ? null : challenge.externalLink?.link}
-              isDisabled={isChallengeLocked}
-              variant={isChallengeLocked ? "outline" : "solid"}
-              isFullWidth
-              isExternal
-            >
-              {builderHasCompletedDependenciesChallenges ? (
-                <chakra.span>{challenge.externalLink.claim}</chakra.span>
-              ) : (
-                <>
-                  <span role="img" aria-label="lock icon">
-                    🔒
-                  </span>
-                  <chakra.span ml={1}>Locked</chakra.span>
-                </>
-              )}
-            </Button>
-            {!builderHasCompletedDependenciesChallenges && (
-              <Tooltip label={lockReasonToolTip}>
-                <IconButton icon={<QuestionOutlineIcon />} />
-              </Tooltip>
+    <Box borderBottom="2px" borderColor={borderColor} backgroundColor={cardBgColor}>
+      <Flex
+        justifyContent="space-between"
+        py={8}
+        ml={14}
+        pl={10}
+        pr={{
+          base: 10,
+          lg: 0,
+        }}
+        borderLeft="8px"
+        borderColor={borderColor}
+        position="relative"
+        direction={{
+          base: "column-reverse",
+          lg: "row",
+        }}
+        _after={
```

Comment:
### Regarding the side timeline line : 
**Approach :** 
For the side timeline line I added a `border-left` to the parent container, Now since the `first` card has only half border I tried hiding the half border with `after` pseudo-selector as suggested [here](https://stackoverflow.com/questions/8804417/css-border-where-only-half-of-a-border-is-visible)
![Screenshot 2022-11-19 at 3 21 18 PM](https://user-images.githubusercontent.com/80153681/202845199-92b88078-50e3-4ee5-a896-7dc1c465130f.jpg)

**Alternative** 
Maybe adding a seprate `div` as a line and positioning it `absolute` to parent. 

I feel adding a left border is the best solution according to  me,  because we are guaranteed that it won't shift on devices and stay consistent on every screen size...but please let me know if we should take any other way 🙌